src/index.ts

```diff
@@ -24,20 +25,26 @@ interface PRDetails {
 }
 
 async function getPRDetails(): Promise<PRDetails> {
-  const { repository, number } = JSON.parse(
-    readFileSync(process.env.GITHUB_EVENT_PATH || "", "utf8")
-  );
-  const prResponse = await octokit.pulls.get({
-    owner: repository.owner.login,
-    repo: repository.name,
-    pull_number: number,
+  const context = github.context;
```

Comment:
The use of `readFileSync` to read the GitHub event path is not necessary since the GitHub context can be used directly to access the pull request details. This improves performance by avoiding file I/O.